app/org/iatoki/judgels/raguel/services/impls/ThreadPostServiceImpl.java

```diff
@@ -83,6 +91,31 @@ public ThreadPost findThreadPostById(long threadPostId) {
     }
 
     @Override
+    public Map<String, Long> getUserJidToPostCountMap(List<String> userJids) {
+        List<UserPostCountModel> postCountModelList = userPostCountDao.getByUserJids(userJids);
+        Set<String> allUser = new HashSet<>(userJids);
```

Comment:
Normally, we use Maps.newHashSet() rather than new HashSet<>(), this way we can track based on methods.